deepmd/fit/ener.py

```diff
@@ -276,8 +276,8 @@ def _build_lower(
     ):
         # cut-out inputs
         inputs_i = tf.slice (inputs,
-                             [ 0, start_index*      self.dim_descrpt],
-                             [-1, natoms* self.dim_descrpt] )
+                             [ 0, start_index, 0],
+                             [-1, natoms, self.dim_descrpt] )
```

Comment:
Looks like descriptors have the same issue, for example
https://github.com/deepmodeling/deepmd-kit/blob/b534355be64f2e819ba38f9793558cee7e5eb912/deepmd/descriptor/se_a.py#L520-L521
https://github.com/denghuilu/deepmd-kit/blob/b534355be64f2e819ba38f9793558cee7e5eb912/deepmd/descriptor/se_a.py#L556-L557
https://github.com/deepmodeling/deepmd-kit/blob/b534355be64f2e819ba38f9793558cee7e5eb912/deepmd/descriptor/se_t.py#L417-L418
https://github.com/deepmodeling/deepmd-kit/blob/b534355be64f2e819ba38f9793558cee7e5eb912/deepmd/descriptor/se_t.py#L445-L446
https://github.com/deepmodeling/deepmd-kit/blob/b534355be64f2e819ba38f9793558cee7e5eb912/deepmd/descriptor/se_t.py#L452-L453
https://github.com/deepmodeling/deepmd-kit/blob/10fdc930d25673ac979f78c8f43148c3fad4d47f/deepmd/descriptor/hybrid.py#L192

I guess `sel` in your model is not big, so you won't catch an exception

Reply:
2,147,483,647/4/1,951,488=275.1
You may test a bigger sum of `sel` like 300